shell/platform/android/test/io/flutter/embedding/engine/loader/FlutterLoaderTest.java

```diff
@@ -31,13 +38,29 @@ public void itReportsUninitializedAfterCreating() {
 
   @Test
   public void itReportsInitializedAfterInitializing() {
-    FlutterInjector.setInstance(new FlutterInjector.Builder().setShouldLoadNative(false).build());
-    FlutterLoader flutterLoader = new FlutterLoader();
+    FlutterJNI mockFlutterJNILoader = mock(FlutterJNI.class);
+    FlutterLoader flutterLoader = new FlutterLoader(mockFlutterJNILoader);
 
     assertFalse(flutterLoader.initialized());
     flutterLoader.startInitialization(RuntimeEnvironment.application);
     flutterLoader.ensureInitializationComplete(RuntimeEnvironment.application, null);
+    shadowOf(getMainLooper()).idle();
     assertTrue(flutterLoader.initialized());
-    FlutterInjector.reset();
+    verify(mockFlutterJNILoader, times(1)).loadLibrary();
+
+    ActivityManager activityManager =
```

Comment:
can this be a separate test? Since it seems unrelated to the test above.

Reply:
Done